.travis.yml

```diff
@@ -1,10 +1,11 @@
 sudo: false
 language: julia
 julia:
- - 0.6
+ - 1.0
```

Comment:
This also drops support to Julia 0.7 @DilumAluthge  @davidavdav

Reply:
I’ll make a new PR that restores Julia 0.7 support!